services/order.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket
+
+
+def create_order(tickets: list, username: str, date: datetime = None) -> None:
+    with transaction.atomic():
+        order = Order.objects.create(
+            user=get_user_model().objects.get(username=username)
+        )
+
+        if date:
+            order.created_at = date
+            order.save()
+
+        for ticket in tickets:
+            Ticket.objects.create(
+                row=ticket["row"],
+                seat=ticket["seat"],
+                movie_session_id=ticket["movie_session"],
+                order=order
+            )
+
+
+def get_orders(username: str = None) -> QuerySet:
+    queryset = Order.objects.all()
+
+    if username:
+        queryset = queryset.filter(user__username=username)
+
+    return queryset.order_by("-user__username")
```

Comment:
Ordering should not be applied here.
There was a bug that made tests fail on some students' computers. It should be fixed by now. Either way, it shouldn't affect tests on GitHub.